docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -514,10 +514,23 @@ composer config extra.runtime.error_handler "\\Ibexa\\Contracts\\Core\\MVC\\Symf
 composer dump-autoload
 ```
 
+### Ibexa Cloud
+
+Update Platform.sh configuration for PHP 8.3.
+
+Generate new configuration with the following command:
+
+```bash
+composer ibexa:setup --platformsh
+```
+
+Review the changes applied to `.platform.app.yaml` and `.platform/`,
+merge with your custom settings if needed, and commit them to Git.
+
```

Comment:
To avoid having wrong VCL, instead of re-generating everything, one could just edit manually for PHP.

````suggestion
### Ibexa Cloud

Update Platform.sh configuration for PHP 8.3.

Edit `.platform.app.yaml` and increase PHP version to 8.3 in the application type:

```diff
- type: php:8.1
+ type: php:8.3
```

````